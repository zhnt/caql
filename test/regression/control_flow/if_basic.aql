// Basic if statement test
let x = 10

if x > 5 {
    print("x is greater than 5")
}

print("done")
