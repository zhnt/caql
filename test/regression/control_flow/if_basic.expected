x is greater than 5
done
